### DISCUSSION TITLE
Customizing the name when exporting the result of a build

### DISCUSSION THREAD

**Discussion by helpermethod** [jbangdev/jbang#1382] [Q&A] [answered]
Is there some way to set the preferred name when exporting a JAR / native-image? E.g. running

    jbang export local --native p2e

will export a native-image named p2e.bin. But what I would like to do is something like p2e-{{platform}}. Is this currently possible?

**Comment by quintesse** [comment, marked as answer]
Yes, you can use the `--output` option.

**Comment by helpermethod** [comment]
Thanks! Where can I find this in the documentation / jbang help?

**Reply by maxandersen** [reply]
`jbang export local --help` ?

**Reply by helpermethod** [reply]
Ah sorry, it seems I was using an old version of JBang. For anyone trying the same thing, this is what I ended with

    jbang export local --native --output p2e-mac p2e
